### DISCUSSION TITLE
How to remove bootstrap from nuxt 3

### DISCUSSION THREAD

**Discussion by vondutch2560** [nuxt/nuxt#26122] [Questions] [answered]
Sorry if my English is a bit poor.
I'm starting to explore Nuxt 3 again, and when running the demo, I noticed that Nuxt 3 added Bootstrap using CDN. I don't use Bootstrap and couldn't find a way to remove it from the project. Previously, I also used Nuxt 3 for mini projects and never encountered this issue

Thank you everyone for reading, hoping to receive your assistance

**Comment by jwhazel** [comment, marked as answer]
Which demo are you looking at? The official demo does not utilize Bootstrap https://stackblitz.com/github/nuxt/starter/tree/v3/?file=package.json

**Reply by vondutch2560** [reply]
Sorry, my mistake. One of my extensions added Bootstrap on all websites. I tried using Ingonito mode and everything is working fine now. Thank you for your assistance
